### DISCUSSION TITLE
How can I catch ERR_CONNECTION_REFUSED errors?

### DISCUSSION THREAD

**Discussion by geauser** [urql-graphql/urql#3135] [Q&A] [answered]
Hi,

I'm trying to catch `ERR_CONNECTION_REFUSED` errors when the target server is offline so I can display an error message on the UI. But I can't seem to catch it using `mapExchange({ onError() { /*...*/ } })`, am I doing something wrong?

Here is the error I get in the console:

![image](https://user-images.githubusercontent.com/48527945/230369860-445a8445-b354-4920-a034-26b3ffcecc5b.png)

**Comment by kitten** [comment, marked as answer]
This should be fixed in `@urql/core@4.0.3` https://github.com/urql-graphql/urql/releases/tag/%40urql/core%404.0.3

This was a regression in v4.

**Reply by geauser** [reply]
Thank you for the quick response, however I'm using `@urql/vue`, what should I do?

**Reply by kitten** [reply]
Just upgrade `@urql/core`, it's a dependency of any binding, including `@urql/vue`.

**Reply by geauser** [reply]
Thank you very much! @kitten
